communication/README.md

```diff
@@ -41,11 +41,14 @@ please [file an issue].
 
 ## Mailing lists
 
-Development announcements and discussions appear on the Google group
-[kubernetes-dev] (send mail to `kubernetes-dev@googlegroups.com`).
+Kubernetes mailing lists are hosted through Gougle Groups.  To
+receive these lists' emails,
+[join](https://support.google.com/groups/answer/1067205) the groups
+relevant to you, as you would any other Google Group.
 
-Users trade notes on the Google group
-[kubernetes-users] (send mail to `kubernetes-users@googlegroups.com`).
+* [kubernetes-dev] hosts development announcements and discussions around developing kubernetes itself
```

Comment:
kubernetes-announce is also an important one

Reply:
added